Tests/DataStructuresTests/BimapTests.swift

```diff
@@ -70,4 +70,20 @@ class BimapTests: XCTestCase {
         XCTAssertEqual(bimap.count, 0)
         XCTAssert(bimap.isEmpty)
     }
+    
+    func testCompose() {
+        let bimap1: Bimap = [0: "a", 2: "b"]
+        let bimap2: Bimap = ["b": 35, "c": 200]
+        let bimap3 = Bimap.compose(bimap1, bimap2)
+        XCTAssertEqual(bimap3[2], 35)
```

Comment:
Would it be simpler to run a comparison between an expected `Bimap` and the actual, such as:

```Swift
XCTAssertEqual(bimap3, [2: 35])
```

Just to make sure there's no other funny business?